src/io.c

```diff
@@ -205,14 +199,16 @@ enum {
 	DISPATCH_IOCNTL_MAX_PENDING_IO_REQS,
 };
 
-static struct dispatch_io_defaults_s {
+extern struct dispatch_io_defaults_s {
```

Comment:
Hmm, don't think that the `extern` is needed here as the statement is a definition.